Mantle/MTLJSONAdapter.m

```diff
@@ -370,13 +370,9 @@ + (NSDictionary *)valueTransformersForModelClass:(Class)modelClass {
 	for (NSString *key in [modelClass propertyKeys]) {
 		SEL selector = MTLSelectorWithKeyPattern(key, "JSONTransformer");
 		if ([modelClass respondsToSelector:selector]) {
-			NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:[modelClass methodSignatureForSelector:selector]];
-			invocation.target = modelClass;
-			invocation.selector = selector;
-			[invocation invoke];
-
-			__unsafe_unretained id transformer = nil;
-			[invocation getReturnValue:&transformer];
+			IMP imp = [modelClass methodForSelector:selector];
+			NSValueTransformer*(*function)(id, SEL) = (void *)imp;
+			__unsafe_unretained NSValueTransformer *transformer = function(modelClass, selector);
```

Comment:
Unfortunately we cannot specify an ARC lifetime qualifier on the caller-side of a function invocation. I can remove the `__unsafe_unretained` on the static variable holding the returned value, however I think it's correct as is.

The called method should return via ARC, transformer as an autoreleased value. Then, if transformer is non-nil, it is immediately strongly retained by the dictionary `result`.